src/test/ctl_cow/ctl_cow.c

```diff
@@ -136,13 +136,26 @@ test_log(const char *path)
 	pmemlog_close(plp);
 }
 
+static void
+test_dax(const char *path)
+{
+	PMEMobjpool *pop = pmemobj_open(path, NULL);
+
+	if (pop == NULL) {
+		UT_FATAL("!cannot open %s", path);
+
```

Comment:
if there's one single-line statement in both branches then curly braces are not needed.